pkg/distributor/ha_tracker_http.go

```diff
@@ -0,0 +1,81 @@
+package distributor
+
+import (
+	"html/template"
+	"net/http"
+	"strings"
+	"time"
+)
+
+const trackerTpl = `
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Cortex HA Tracker Status</title>
+	</head>
+	<body>
+		<h1>Cortex HA Tracker Status</h1>
+		<p>Current time: {{ .Now }}</p>
+		<form action="" method="POST">
+			<input type="hidden" name="csrf_token" value="$__CSRF_TOKEN_PLACEHOLDER__">
```

Comment:
This is only needed if the page has an "action" - ie a button.  Same with the form.